Assets/Scripts/Menu Scripts/DroneMenu.cs

```diff
@@ -0,0 +1,67 @@
+﻿using ISAACS;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+
+public class DroneMenu : MonoBehaviour {
+
+    public Text dronePosText;
+    public Text droneNameText;
+    public Text droneAuthorityText;
+    GameObject menuCanvas;
+    double homeLat;
+    double homeLong;
+    bool hasAuthority;
+
+    // Use this for initialization
+    void Start () {
+        //f = GameObject.Find("DronePos");
+        //Debug.Log(f);
+        menuCanvas = GameObject.Find("DroneMenuCanvas");
```

Comment:
This will be a problem with multiple drones. One way is to simply use GamObjectFindInChildren but there's other better ways to do it too.